scripts/verify-contracts.ts

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs')
```

Comment:
Please run prettier on this file too.....the inconsistent indenting (like lines 13 and 14) makes things very hard to read